larky/src/main/resources/vendor/Crypto/Util/asn1.star

```diff
@@ -867,7 +867,9 @@ def DerObjectId(value='', implicit=None, explicit=None):
         Raises:
             ValueError: in case of parsing errors.
         """
-
+        if type(der_encoded) == 'string':
```

Comment:
@OmegaPointZero why was this put here? Was the DerObjectId not encoded property when it came in under decode()?